lib/pg_ltree/ltree.rb

```diff
@@ -16,6 +16,7 @@ def ltree(column = :path, options: { cascade: true })
 
       if options[:cascade]
         after_update :cascade_update
+        before_destroy :delete_ltree_column_value
```

Comment:
I've added a test per your request and moved the conditional to use callback `if:`.